File: GTTS/INL.ReferenceService/Models/BusinessUnits_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.ReferenceService.Models
{
    public class BusinessUnits_Item
	{
		public long BusinessUnitID { get; set; }
		public int BusinessUnitTypeID { get; set; }
		public string BusinessUnit { get; set; }
		public string Acronym { get; set; }
		public long? BusinessUnitParentID { get; set; }
		public long? UnitLibraryUnitID { get; set; }
        public int? PostID { get; set; }
        public string Description { get; set; }
		public bool IsActive { get; set; }
		public bool IsDeleted { get; set; }
		public bool HasPrograms { get; set; }
		public string LogoFileName { get; set; }
		public string VettingPrefix { get; set; }
		public bool hasDutyToInform { get; set; }
		public int ModifiedByAppUserID { get; set; }
		public DateTime ModifiedDate { get; set; }
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.ReferenceService.Models/business-units_item.ts



export class BusinessUnits_Item  {
  
	public BusinessUnitID: number = 0;
	public BusinessUnitTypeID: number = 0;
	public BusinessUnit: string = "";
	public Acronym: string = "";
	public BusinessUnitParentID?: number;
	public UnitLibraryUnitID?: number;
	public PostID?: number;
	public Description: string = "";
	public IsActive: boolean = false;
	public IsDeleted: boolean = false;
	public HasPrograms: boolean = false;
	public LogoFileName: string = "";
	public VettingPrefix: string = "";
	public hasDutyToInform: boolean = false;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}


